### ISSUE TITLE
Could not pass event EntityDamageByEntityEvent

### ISSUE THREAD

**Issue by Yselith** [TheFantasyCraft/Forge-Permittor#12]
Hello,

I get console spam :

http://pastebin.com/cx82wZN7

**Comment by thomas15v** [comment]
Oh that, Yeah I also found that when I tried it with worldguard 6.0. The only way to fix this is recompile the plugin with 6.0. you will have no errors if you do. Only it will not work on lower versions any-more.

to do that download git and maven.
and version https://github.com/TheFantasyCraft/Forge-Permittor/blob/master/pom.xml#L51 to '6.0.0-SNAPSHOT'

than do these commands to recompile the plugin
git clone https://github.com/TheFantasyCraft/Forge-Permittor
cd Forge-Permittor
mvn install
